andrew_eye_stream_data.py

```diff
@@ -0,0 +1,50 @@
+from talon import app, ui
+from talon_plugins.eye_mouse import tracker
```

Comment:
@zkirby How did you find out about these API members? I took a look at the docs and it looks like they're not documented, at least as far as I can find. Did you find these in the source code?

Reply:
Yeah it was fishing around in the source code